contracts/prize-strategy/PeriodicPrizeStrategy.sol

```diff
@@ -270,6 +271,16 @@ abstract contract PeriodicPrizeStrategy is Initializable,
   function _awardTickets(address user, uint256 amount) internal {
     prizePool.award(user, amount, address(ticket));
   }
+  
+  /// @notice Awards collateral as sponsorship to a user
+  /// @param user The user to whom the tickets are minted
+  /// @param amount The amount of interest to mint as tickets.
```

Comment:
Missing natspec for `tokenIndex`

Reply:
Added natspec and updated `tokenType` to `tokenIndex` in other contracts to adhere to a single variable name.